controllers/front/callbacknotification.php

```diff
@@ -35,104 +35,104 @@ public function postProcess()
-                    Db::getInstance()->Execute($sql);
-                    $payment = $order->getOrderPaymentCollection();
-                    if (isset($payment[0])) {
-                        $payment[0]->transaction_id = pSQL($shopOrderId);
-                        $payment[0]->save();
+                        Db::getInstance()->Execute($sql);
+                        $payment = $order->getOrderPaymentCollection();
+                        if (isset($payment[0])) {
+                            $payment[0]->transaction_id = pSQL($shopOrderId);
+                            $payment[0]->save();
+                        }
+                        $this->unlock($fp);
+                        exit('Order status updated to Accepted');
+                    } else {
+                        // Unexpected scenario
+                        $mNa = $this->module->name;
+                        PrestaShopLogger::addLog('Unexpected scenario: Callback notification was received for Transaction '
+                                                 . $shopOrderId . ' with payment status ' . $transactionStatus, 3, '1005', $mNa,
+                            $this->module->id, true);
+                        $this->unlock($fp);
+                        exit('Unrecognized status received ' . $transactionStatus);
                     }
-                    $this->unlock($fp);
-                    exit('Order status updated to Accepted');
-                } elseif ($transactionStatus === 'epayment_declined') {
-                    // Update to Payment Error
-                    $order->setCurrentState((int) Configuration::get('PS_OS_ERROR'));
-                    // Update payment status to 'declined'
-                    $sql = 'UPDATE `' . _DB_PREFIX_ . 'altapay_order` 
-                    SET `paymentStatus` = \'declined\' WHERE `id_order` = ' . $order->id;
-                    Db::getInstance()->Execute($sql);
-                    $this->unlock($fp);
-                    exit('Order status updated to Error');
-                } else {
-                    // Unexpected scenario
-                    $mNa = $this->module->name;
-                    PrestaShopLogger::addLog('Unexpected scenario: Callback notification was received for Transaction '
-                                   . $shopOrderId . ' with payment status ' . $transactionStatus, 3, '1005', $mNa,
-                        $this->module->id, true);
-                    $this->unlock($fp);
-                    exit('Unrecognized status received ' . $transactionStatus);
                 }
+            } else {
+                // Unexpected scenario
+                $mNa = $this->module->name;
+                PrestaShopLogger::addLog('Callback notification was received for Transaction '
+                                         . $shopOrderId . ' with payment status ' . $transactionStatus, 3, '1005', $mNa,
+                    $this->module->id, true);
```

Comment:
Confusing line breaks
```suggestion
                PrestaShopLogger::addLog(
                    'Callback notification was received for Transaction ' . $shopOrderId . ' with payment status ' . $transactionStatus,
                    3,
                    '1005',
                    $mNa,
                    $this->module->id,
                    true
                );
```